### ISSUE TITLE
Splash Screens

### ISSUE THREAD

**Issue by cambragol** [alexbatalov/fallout2-ce#389]
I may just be missing something, but are splash screens not supported? Or does the game just load so fast that there is no need for a splash screen?

It would be nice to see them again, as they add to the vibe of the game.

**Comment by alexbatalov** [comment]
I've arranged a fix for this one, but we need to check other platforms before merging.

**Comment by alexbatalov** [comment]
You're right, current fix seems to be unreliable. It only works if the game starts with macOS permission warning.

**Comment by alexbatalov** [comment]
@cambragol Can you please validate the last attempt? Works for me both with debugger attached and standalone.
